public/gallery.html

```diff
@@ -4,9 +4,11 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<title>Pet Gallery</title>
+		<link rel="stylesheet" href="CSS/gallery.css">
```

Comment:
CSS is cascading. So if you first put this and then tailwind you risk having things overwritten by it